contracts/BSM/Investors/AMM/AMMInvestorsHelper.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.13;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@uniswap/v2-periphery/contracts/interfaces/IUniswapV2Router02.sol";
+import "./interfaces/ISovrynSwapNetwork.sol";
+
+contract AMMInvestorsHelper is OwnableUpgradeable, ReentrancyGuard {
+    using SafeERC20 for IERC20;
+
+    enum RouterTypes {
+        UniswapV2,
+        Sovryn
+    }
+
+    /* ========== INITIALIZER ========== */
+
+    function initialize() external initializer {
+        __Ownable_init();
+
+        require(owner() != address(0), "AMMInvestorsHelper: owner must be set");
```

Comment:
```suggestion
```

__Ownable_init() takes care of that, isn't it?

Reply:
Nice!